### PR TITLE
Replace wget in instructions with curl, as it's more widely avail. on macs

### PR DESCRIPTION
**Pull request by paulbellamy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Interact with your containers directly: pause, restart and stop containers. Laun
 ## <a name="getting-started"></a>Getting started
 
 ```
-sudo wget -O /usr/local/bin/scope https://git.io/scope
+sudo curl -L git.io/scope -o /usr/local/bin/scope
```
Comment (tomwilkie) [inline README.md]: Will this default to https?

Comment (paulbellamy) [inline README.md]: No. It redirects http -> https -> script.

Comment (tomwilkie) [inline README.md]: I guess thats fine?  I mean, if you're downloading a shell script as root, https isn't exactly going to save you...

Comment (rade) [inline README.md]: no https:// prefix is fine by me.

Comment (paulbellamy) [inline README.md]: We don't use it on weave, so...

**Comment by tomwilkie** [comment]
LGTM

**Comment by mjlodge** [comment]
We should also make this change on the website...

**Comment by pidster** [comment]
@mjlodge already done.

**Pull request merged**